From 31abfa4adce288b4e5aac37307ff5f0e63b6f590 Mon Sep 17 00:00:00 2001
From: Vincent Chernin <38842733+vchernin@users.noreply.github.com>
Date: Fri, 6 Aug 2021 15:06:02 -0700
Subject: [PATCH] About menu icon fix. The value of the icon name in the about menu is hardcoded to the Flatpak App ID so the icon loads.
This is the simplest solution I could come up with, as the .desktop file itself wants the Flatpak App ID as well, it's easier to be consistent everywhere.
One could also try making multiple copies of the icon, or symlinks, but that seemed unnecessarily complex.

---
 src/application.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/application.cpp b/src/application.cpp
index a4961ac4..7d618bce 100644
--- a/src/application.cpp
+++ b/src/application.cpp
@@ -353,7 +353,7 @@ void Application::create_actions() {
     dialog->set_version(VERSION);
     dialog->set_comments(_("Audio effects for PipeWire applications"));
     dialog->set_authors({"Wellington Wallace"});
-    dialog->set_logo_icon_name("easyeffects");
+    dialog->set_logo_icon_name("com.github.wwmm.easyeffects");
     dialog->set_license_type(Gtk::License::GPL_3_0);
     dialog->set_website("https://github.com/wwmm/pulseeffects");
 
